.github/workflows/xrefcheck-validate-markdown-links.yml

```diff
@@ -0,0 +1,16 @@
+name: Validate Markdown links
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  xrefcheck:
+    name: Check links
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: serokell/xrefcheck-action@v1
```

Comment:
This has very few stars after two years. Many stars doesn't mean significant usage, but few stars might be a signal of limited usage.

So just out of caution: is this the right tool for this job?

Reply:
Totally open to suggestions. @jrhemstad adopted this for Thrust (or some other project?) so I was following that lead. Also I am having trouble making this work, so I’d like another choice. 😅 Do you have alternatives in mind?

Reply:
Maybe this: https://github.com/marketplace/actions/markdown-link-check

Reply:
I recall looking into several options and ended up using `xrefcheck` because it was the only one I could find that supported relative Markdown links. I don't recall if I came across https://github.com/marketplace/actions/markdown-link-check, but it'd be worth checking out. 

I otherwise have no special attachment to `xrefcheck` and would be happy to use something else.